### PR TITLE
Resolution for Issues #1044 & #1007

### PR DESCRIPTION
**Pull request by 1lluzi0nzz**
Hello, I think this PR should close out a couple issues in the project as well as present albums in a cleaner way:

Tray & Taskbar Icons:
pr1

Album Year Display:
pr2

Full Notes:
- Album Year instead of Timestamp

- Fixed Typo in the equalizer presets

- Issue with the Taskbar Icon (#1044) should be resolved (Verified on Windows)

- Issue with the Tray Icon (#1007) not working correctly on release builds should be resolved (Verified on Windows)

### CODE REVIEW

**Comment by jreiser1** [comment]
There's a ton of commits mainly because we were messing with our configuration stuff, setting up our own CI, and making sure that our changes wouldn't interfere with your configuration. We would be happy to squash these down if you would like.

**Comment by nukeop** [comment]
Thanks for contributing. Don't worry about the number of commits, I'll just squash them. I'll run the CI now and will review it soon.

File: .gitignore [hunk 1]
```diff
@@ -73,4 +73,13 @@ lerna-debug.log
...
-build-dir
\ No newline at end of file
+build-dir
+
+#Travis/IDE stuff
+.travis.yml
+.vs/
+
+#package-lock
+package-lock.json
```
Comment (nukeop) [inline .gitignore:83]: Please don't commit these (package-lock and package.json) as they need to be update if we install new packages.

File: packages/main/src/services/config/index.ts [hunk 2]
```diff
@@ -39,7 +39,8 @@ class Config {
     this.youtubeSearch = 'https://www.googleapis.com/youtube/v3/search?part=id,snippet&type=video&maxResults=50&q=';
     this.supportedFormats = _.uniq(pkg.build.fileAssociations.map(({ ext }) => ext));
 
-    const iconPath = path.resolve(__dirname, this.isProd() ? 'resources' : '../resources/media');
+    // const iconPath = path.resolve(__dirname, this.isProd() ? 'resources' : '../resources/media');
+    const iconPath = path.resolve(__dirname, this.isProd() ? '../../../resources/media' : '../resources/media');
```
Comment (nukeop) [inline packages/main/src/services/config/index.ts:43]: Thanks for getting to the bottom of this. Could you please delete the commented out line?

File: .gitignore (hunk 1 above)
Comment (1lluzi0nzz) [reply .gitignore]: These lines have been taken out of .gitignore, should be tracked again.

Thanks!

File: packages/main/src/services/config/index.ts (hunk 2 above)
Comment (1lluzi0nzz) [reply packages/main/src/services/config/index.ts:42]: I removed that comment.

Thanks for reviewing this.

**Comment by nukeop** [comment]
Thanks for solving this!

**Pull request merged**